UseLatexMk.cmake

```diff
@@ -101,6 +101,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
+include_guard(GLOBAL)
```

Comment:
This is CMake 3.10 (which I am fine to require)

Reply:
Thanks for review!  
I'll add another commit to increase the CMake requirement to 3.10? Alternately, we could enclose this line with a CMake version check. It is up to you to decide.

Reply:
I am not very conservative here: People that use this project will typically copy and paste it anyway, so not much harm to be done. In fact, the file itself could use a cmake version check, because the one in CMakeLists.txt is not really used in the C&P case.

Reply:
I added an according commit to this pull request.